.ci/jsoref-spellchecker/whitelist.words

```diff
@@ -605,7 +605,7 @@ JCheck
 JCombo
 JComponent
 jcp
-JDBC
+jdbc
```

Comment:
@jsoref Can you explain the benefit of having uppercase words in whitelist if we have to eventually replace them with lowercase when they are used in that form?

Reply:
Hmm, for this use case, that's probably correct. I might have time tonight (if not, probably Sunday).

Reply:
> for this use case, that's probably correct.

Is there some use case where this behavior is preferred?

Reply:
`JavaScript` is a correct spelling, `Java` is a correct spelling, `script` is a word, `javascript` can be a correct thing in circumstances.

`Javascript` is actually a misspelling.

There are a couple of edges like this, but, in general it probably isn't worth worrying about them. And, since I already cut out the code that shows case variations, you could easily miss them depending on order.